src/com/cdd/bao/editor/DetailPane.java

```diff
@@ -874,7 +879,7 @@ private void pressedAnnotationButton(Schema.Assignment assn, int idx)
 	}
 
 	// respond to focus so that one of the blocks gets a highlight
-	private void observeFocus(Control field, final int idx)
+	public void observeFocus(Control field, final int idx)
```

Comment:
Out of curiosity, what is the reason to change the visibility from `private` to `public`?

Reply:
Ignore that question